Makefile

```diff
@@ -1,10 +1,19 @@
+export GO111MODULE := on
+
 all: test
 
 install-deps:
-	GO111MODULE=on go get
+	go get
+.PHONY: install-deps
```

Comment:
Oh man, I have to admit I had no idea why `.PHONY` targets were used until RTFMing just now.

OOC, is there a reason to place the phony target below rather than above the body? Just a stylistic thing? (Asking because I noticed the manual example places above.

Reply:
No other reason than me realizing we were missing `.PHONY` targets when I looked at Goreleaser's Makefile and reusing their convention: https://github.com/goreleaser/goreleaser/blob/master/Makefile. If placing the annotation above the body is more common I'm happy to update the PR accordingly!

Reply:
Ah, got it. Thanks!

> If placing the annotation above the body is more common I'm happy to update the PR accordingly!

Don't really mind either way, so whatever works.